source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -90,21 +86,18 @@ class NiRFmxWLANDriverApiTests : public Test {
   std::unique_ptr<NiRFmxWLAN::Stub> stub_;
 };
 
-void GetCommaSeparatedStringFromArray(char* array_of_names, int num_of_names, int name_length, char* comma_seperated_name)
+void GetCommaSeparatedStringFromArray(const std::vector<std::string>& array_of_names, std::string& comma_seperated_name)
 {
   int name_index = 0;
   int comma_index = 0;
-  while (name_index < num_of_names) {
-    int char_index;
-    for (char_index = 0; char_index < name_length; ++char_index) {
-      if (*(array_of_names + name_index * name_length + char_index) == '\0') break;
-      comma_seperated_name[comma_index++] = *(array_of_names + name_index * name_length + char_index);
-    }
-    if (++name_index != num_of_names) {
-      comma_seperated_name[comma_index++] = ',';
+  std::stringstream output;
+  while (name_index < array_of_names.size()) {
+    output << array_of_names[name_index];
+    if (++name_index != array_of_names.size()) {
+      output << ',';
     }
   }
-  comma_seperated_name[comma_index] = '\0';
+  comma_seperated_name = output.str();
```

Comment:
Looks like `comma_seperated_name` can be a return value.

Reply:
Good point. Changed to return output.str() and not pass the return string in.